iFixit/react-commerce
# Refactoring bit table layout

This PR introduce an even more compact design proposal for the bit table section.
In this reviewed design we can merge bits from various screwdriversize in the same row, but still preserving readability.
Bit types without size variations are pushed to the end of the list to keep the more dense information in the top left corner.

### QA

1. Visit Vercel preview
2. Verify that the new implementation is aligned with the updated mockup visible [here](https://www.figma.com/file/BolmFORO6ZfHXePOSyEGSS/iFixit---Page-sections?node-id=1766%3A10707&mode=dev)
